buildSrc/src/main/java/com/osacky/flank/gradle/YamlWriter.kt

```diff
@@ -106,15 +106,21 @@ internal class YamlWriter {
   }
 
   @VisibleForTesting
-  internal fun createDeviceString(devices: List<Device>): String = buildString {
+  internal fun createDeviceString(devices: List<Map<String, String?>>): String = buildString {
```

Comment:
About to merge, but I need to talk this out. :) Is it possible for the value to be `null` here? For example if we have `orientation` in the map, the value for `orientation` must be non-null. Same goes for `model`?

Reply:
You know what, that's a good point haha.  I'll remove the `?` that's unnecessary

Reply:
Fixed :) 8b6369454c00fadc2f9160668f089d8c7888f393